### ISSUE TITLE
Build outstanding requests page

### ISSUE THREAD

**Issue by kellyChex** [UCSCLibrary/dams_project_mgmt#219]

**Comment by JessWaggs** [comment]
The outstanding requests page looks like it is in Aeon rather than the DAMS. I can grab a screenshot of this page in our Aeon system.

**Comment by JessWaggs** [comment]
It feels odd to me that the user would view Authorized works in DAMS but outstanding requests in Aeon? Seems optimal to have these in the same system, if possible.

**Comment by kellyChex** [comment]
@JessWaggs Yes, Heather mentioned the Outstanding Requests page is what comes from Aeon. It looked like DAMs with an Aeon page. @NedHenry Does that seem right or wrong from your knowledge?
